src/CSnakes.Runtime.Tests/Python/ImportTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿
+using CSnakes.Runtime.Python;
+
+namespace CSnakes.Runtime.Tests.Python;
+public class ImportTests : RuntimeTestBase
+{
+    [Fact]
+    public void TestImportModule()
+    {
+        using (GIL.Acquire())
+        {
+            using PyObject? sys = Import.ImportModule("sys");
+            Assert.NotNull(sys);
+            Assert.Equal("<module 'sys' (built-in)>", sys!.ToString());
```

Comment:
The `!` isn't needed since [`Assert.NotNull` narrows its argument to being non-nullable](https://github.com/xunit/assert.xunit/blob/cac8b688c193c0f244a0bedf3bb60feeb32d377a/NullAsserts.cs#L31):

```suggestion
            Assert.Equal("<module 'sys' (built-in)>", sys.ToString());
```